site/src/api/typesGenerated.ts

```diff
@@ -1908,3 +1908,92 @@ export const WorkspaceTransitions: WorkspaceTransition[] = [
 
 // From codersdk/workspaceproxy.go
 export type RegionTypes = Region | WorkspaceProxy
+
+// Code generated by 'make site/src/api/typesGenerated.ts'. DO NOT EDIT.
+
+// From healthcheck/accessurl.go
+export interface HealthcheckAccessURLReport {
+  readonly access_url: string
+  readonly healthy: boolean
+  readonly reachable: boolean
+  readonly status_code: number
+  readonly healthz_response: string
+  readonly error?: string
+}
+
+// From healthcheck/derp.go
+export interface HealthcheckDERPNodeReport {
+  readonly healthy: boolean
+  // Named type "tailscale.com/tailcfg.DERPNode" unknown, using "any"
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- External type
+  readonly node?: any
+  // Named type "tailscale.com/derp.ServerInfoMessage" unknown, using "any"
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- External type
+  readonly node_info: any
+  readonly can_exchange_messages: boolean
+  // This is likely an enum in an external package ("time.Duration")
```

Comment:
Add time.Duration to this list:

https://github.com/coder/coder/blob/a673c2bbcfa719f9d01750d7f97753cf4babf44f/scripts/apitypings/main.go#L759-L788

(we should make this a config file instead of hard coded)